server/src/controllers/session.ts

```diff
@@ -8,7 +9,8 @@ router.post('/', async (req, res) => {
     const room = rooms.createRoom();
     res.json({ room });
   } catch (ex) {
-    console.error(ex);
+    res.json(ex);
+    log.info(ex);
```

Comment:
Jeżeli ten catch bedzie uruchomiony to znaczy ze coś sie sypnelo na etapie tworzenia pokoju, nie logowałbym tego raczej na poziomie 'info' tylko bardziej jako 'error'